docs/en/setup/service-agent/java-agent/Supported-list.md

```diff
@@ -81,6 +81,7 @@
 * [Hystrix: Latency and Fault Tolerance for Distributed Systems](https://github.com/Netflix/Hystrix) 1.4.20 -> 1.5.12
 * Scheduler
   * [Elastic Job](https://github.com/elasticjob/elastic-job) 2.x
+  * [Apache Elastic Job](https://github.com/apache/shardingsphere-elasticjob) 3.0.0-alpha
```

Comment:
Sorry, `Apache Elastic Job`? Maybe `Apache ShardingSphere-Elasticjob` is better?

Reply:
had update, please recheck